.circleci/config.yml

```diff
@@ -330,7 +330,7 @@ commands: # reusable commands with parameters
             taskname=$(echo $CIRCLE_JOB | sed -e 's/-\(postgres\|oracle\|[0-9]\).*//')
             bundle exec rake "test:files:${taskname}" | circleci tests run --command="cat > $list" --verbose --split-by=timings
             if [ -s $list ]; then
-              bundle exec rake test:run TEST="$(cat $list)" TESTOPTS=--verbose --verbose --trace
+              bundle exec rails test $(cat $list | awk 'ORS=" "') --verbose --backtrace
```

Comment:
`awk 'ORS=" "'` is to merge the list of tests from the file that contain one path per line into a single line separated by spaces.

Reply:
Isn't it nicer to see multi-line?

btw why change to `rails`?

Reply:
A little bit better way
```suggestion
              bundle exec rails test $(awk 'ORS=" " < $list') --verbose --backtrace
```

Reply:
Why is this better?

Reply:
https://porkmail.org/era/unix/award

It doesn't make real difference in this case but is not advantageous, not easier to read and not cool :angel: